CMakeLists.txt

```diff
@@ -624,7 +624,7 @@ if(CONFIG_TIMEOUT_64BIT)
   set(SYSCALL_SPLIT_TIMEOUT_ARG --split-type k_timeout_t)
 endif()
 
-if(CONFIG_BOARD_SCORPIO)
+if(CONFIG_BOARD_SCORPIO AND NOT CONFIG_RISCV_MMU)
```

Comment:
I didnt know you that you can do `AND NOT` :)

Reply:
Yup! 😄